lib/utils/logger/log_settings.js

```diff
@@ -1,103 +1,106 @@
-const lodashClone = require('lodash.clone');
-
-const Settings = {
-  outputEnabled: true,
-  showResponseHeaders: false,
-  showRequestData: {
-    enabled: true,
-    trimLongScripts: true
-  },
-  detailedOutput: true,
-  disableErrorLog: false,
-  log_timestamp: false,
-  timestamp_format: null,
-  enabled: true
-};
-
-module.exports = new (function() {
-  const logSettings = lodashClone(Settings, true);
-
-  class LogSettings {
-    get outputEnabled() {
-      return logSettings.outputEnabled;
-    }
-
-    get detailedOutput() {
-      return logSettings.detailedOutput;
-    }
+class LogSettings {
+  #outputEnabled;
+  #showResponseHeaders;
+  #showRequestData;
+  #detailedOutput;
+  #disableErrorLog;
+  #log_timestamp;
+  #timestamp_format;
+  #enabled;
+  #htmlReporterEnabled;
```

Comment:
we don't need this, since we are declaring private class members in the constructor itself

Reply:
These are required to use `#` identifier. You can check the MDN doc you shared as well.